src/source/Signaling/LwsApiCalls.c

```diff
@@ -12,6 +13,107 @@ VOID lwsSignalHandler(INT32 signal)
     gInterruptedFlagBySignalHandler = TRUE;
 }
 
+static SignalingRole_t getChannelRoleType(SIGNALING_CHANNEL_ROLE_TYPE roleType)
+{
+    SignalingRole_t ret = SIGNALING_ROLE_NONE;
+
+    switch (roleType) {
+        case SIGNALING_CHANNEL_ROLE_TYPE_MASTER:
+            ret = SIGNALING_ROLE_MASTER;
+            break;
+        case SIGNALING_CHANNEL_ROLE_TYPE_VIEWER:
+            ret = SIGNALING_ROLE_VIEWER;
+            break;
+        default:
+            ret = SIGNALING_ROLE_NONE;
+            break;
+    }
+
+    return ret;
+}
+
+static STATUS getMessageType(SignalingTypeMessage_t messageType, SIGNALING_MESSAGE_TYPE* pMessageType)
+{
+    ENTERS();
+    STATUS retStatus = STATUS_SUCCESS;
+
+    if (messageType == SIGNALING_TYPE_MESSAGE_SDP_OFFER) {
+        *pMessageType = SIGNALING_MESSAGE_TYPE_OFFER;
+    } else if (messageType == SIGNALING_TYPE_MESSAGE_SDP_ANSWER) {
+        *pMessageType = SIGNALING_MESSAGE_TYPE_ANSWER;
+    } else if (messageType == SIGNALING_TYPE_MESSAGE_ICE_CANDIDATE) {
+        *pMessageType = SIGNALING_MESSAGE_TYPE_ICE_CANDIDATE;
+    } else if (messageType == SIGNALING_TYPE_MESSAGE_GO_AWAY) {
+        *pMessageType = SIGNALING_MESSAGE_TYPE_GO_AWAY;
+    } else if (messageType == SIGNALING_TYPE_MESSAGE_RECONNECT_ICE_SERVER) {
+        *pMessageType = SIGNALING_MESSAGE_TYPE_RECONNECT_ICE_SERVER;
+    } else if (messageType == SIGNALING_TYPE_MESSAGE_STATUS_RESPONSE) {
+        *pMessageType = SIGNALING_MESSAGE_TYPE_STATUS_RESPONSE;
+    } else {
+        *pMessageType = SIGNALING_MESSAGE_TYPE_UNKNOWN;
+        CHK_WARN(FALSE, retStatus, "Unrecognized message type received");
+    }
+
+CleanUp:
+
+    LEAVES();
+    return retStatus;
+}
+
+static STATUS updateIceServerList(PSignalingClient pSignalingClient, SignalingIceServer_t* pIceServers, SIZE_T numIceServers)
+{
+    ENTERS();
+    STATUS retStatus = STATUS_SUCCESS;
+    UINT32 i, j;
+
+    CHK(pIceServers != NULL, STATUS_SIGNALING_ICE_CONFIG_REFRESH_FAILED);
```

Comment:
Are you asking if it's possible that pIceServers is NULL? If it's the question, then no. It's not possible to get NULL because it's a static function it's only used inside LwsApiCalls.c file.